Library/ViewModels/ProjectPamphletViewModel.swift

```diff
@@ -46,7 +46,7 @@ public protocol ProjectPamphletViewModelOutputs {
 
   var goToDeprecatedViewBacking: Signal<BackingData, Never> { get }
 
-  var goToManageViewPledge: Signal<PledgeData, Never> { get }
+  var goToManageViewPledge: Signal<Project, Never> { get }
```

Comment:
This output and related TestObservers really should just be `goToManagePledge`, I think we fixed this naming elsewhere but forgot to update it here 😬

Reply:
I can update it 👍